projects/nge-ide/src/sidebar/sidebar.component.ts

```diff
@@ -76,10 +77,12 @@ export class SidebarComponent implements OnInit, OnDestroy {
     private readonly ideService: IdeService,
     private readonly storageService: StorageService,
     private readonly changeDetectionRef: ChangeDetectorRef,
-    private readonly viewContainerService: ViewContainerService
+    private readonly viewContainerService: ViewContainerService,
+    private readonly settingsService: SettingsService
   ) {}
 
   ngOnInit(): void {
+    this.settingsService.activate();
```

Comment:
Contributions should not be activated manually, this is handled by the ide service upon startup : 

https://github.com/cisstech/nge-ide/blob/main/projects/nge-ide/core/src/ide.service.ts#L82

Reply:
Thanks for the review! I'm fixing it right now.